cdk-v2/Vagrantfile

```diff
@@ -36,8 +36,16 @@ SUBSCRIPTION_INFO = "
     $ export SUB_PASSWORD=password
   "
 
-NO_SERVICE_MANAGER_ERROR = "vagrant-service-manager plugin is not installed, run `vagrant plugin install vagrant-service-manager` to install the plugin."
-NO_VAGRANT_REGISTRATION_ERROR = "vagrant-registration plugin is not installed, run `vagrant plugin install vagrant-registration` to install the plugin."
+REQUIRED_PLUGINS = %w(vagrant-service-manager vagrant-registration)
+message = ->(name) { "#{name} plugin is not installed, run `vagrant plugin install #{name}` to install it." }
+errors = []
+# Validate and collect error message if plugin is not installed
+REQUIRED_PLUGINS.each {|plugin| errors << message.call(plugin) unless Vagrant.has_plugin?(plugin) }
+if errors.size > 1
+  raise errors.inject(''){|msg, e| msg += "\n * #{e}" }
```

Comment:
@hferentschik . Fixed.

Now error messages:
Single Error:
```
Error: vagrant-registration plugin is not installed, run `vagrant plugin install vagrant-registration` to install it.
```
Multiple Error:
```
Errors: 
* vagrant-service-manager plugin is not installed, run `vagrant plugin install vagrant-service-manager` to install it.
* vagrant-registration plugin is not installed, run `vagrant plugin install vagrant-registration` to install it.
```

Putting `Error(s)` was my choice. If it is not correct, let me know.